init_tables.sql

```diff
@@ -10,8 +10,8 @@ CASCADE;
 -- create users table (note: 'user' is a reserved keyword in postgres)
 CREATE TABLE users (
     id serial PRIMARY KEY,
-    email varchar(254) NOT NULL,
-    password varchar(30) NOT NULL,
+    email varchar(254) NOT NULL UNIQUE,
+    password varchar(60) NOT NULL,
```

Comment:
Forgot to mention this in my commit, but the password also needed to be longer/bigger since we'll be storing a hashed password generated by `bcrypt`.